core/src/de/adesso/brainysnake/BrainySnake.java

```diff
@@ -59,17 +59,13 @@
     @Override
     public void create() {
         pixmap = new Pixmap(WIDTH, HEIGHT, Pixmap.Format.RGBA8888);
-        stage = new Stage();
+        mainStage = new Stage();
         createBasicSkin();
-        
-        try {
-            logoTexture = new Texture(Gdx.files.internal("./core/src/de/adesso/brainysnake/img/BrainySnake_Headline.png"));
-            TextureRegion region = new TextureRegion(logoTexture, 0, 0, 629, 180);
-            logoBrainySnake = new Image(region);
-            logoBrainySnake.setPosition(Config.APPLICATION_WIDTH / 4, Config.APPLICATION_HEIGHT - 250);
-        } catch (Exception e){
-            LOGGER.error(e.toString());
-        }
+
+        logoTexture = new Texture("./core/assets/BrainySnake_Headline.png");
```

Comment:
Move the png to the global assets folder
https://github.com/libgdx/libgdx/wiki/Managing-your-assets

Reply:
Done.

Reply:
Meinte den übergeordneten assets Ordner. Wie beim Logo

Reply:
Ich verstehe das gerade nicht 100%. Wurde das nicht in dem Commit (54051b5550dee6dc4933722504f37fd99fe4ac39) erledigt?

Reply:
@FrederikSchlemmer Jup. Das ist erledigt. Evtl. hat Florian noch einen älteren Stand gesehen.